yudongheon/users/views.py

```diff
@@ -0,0 +1,69 @@
+import re
+import bcrypt
+import jwt
+import json
+
+from django.views import View
+from django.http import HttpResponse, JsonResponse
+from .models import User #UserCoupon, Coupon
+
+PASSWORD_LENGTH = 8
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        email = data.get('email', None)
+        password = data.get('password', None)
+        name = data.get('name', None)
+        last_name = data.get('last_name', None)
+        first_name = data.get('first_name', None)
+        phone_number = data.get('phone_number', None)
+        
+        password_validation = re.compile('^[a-zA-Z0-9]*$')
+        email_validation    = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+        
+        hashed_password     = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+        if not email or not password or not name or not last_name or not first_name or not phone_number:
+            return JsonResponse({'message' : 'CHECK_YOUR_INPUT'}, status=400)
+
+        if User.objects.filter(email=email).exists():
+            return JsonResponse({'message' : 'EXITST_EMAIL'}, status=400)
+
+        if not password_validation.match(password):
+            return JsonResponse({'message' : 'NOT_MATCH_PASSWORD_FORM'}, status=400)
+        
+        if not email_validation.match(email):
+            return JsonResponse({'message' : 'NOT_MATCHED_EMAIL_FORM'}, status=400)
+
+        User.objects.create(
+                email           = email,
+                password        = hashed_password,
+                name            = name,
+                last_name       = last_name,
+                first_name      = first_name,
```

Comment:
이건 회원가입시 필수 정보는 아니지 않아요?
```suggestion
                last_name       = last_name,
                first_name      = first_name,
```